test/flows/check_travel_during_coronavirus_flow_test.rb

```diff
@@ -7,6 +7,7 @@ class CheckTravelDuringCoronavirusFlowTest < ActiveSupport::TestCase
   setup do
     testing_flow CheckTravelDuringCoronavirusFlow
     stub_worldwide_api_has_locations(%w[spain ireland italy poland])
+    @calculator = SmartAnswer::Calculators::CheckTravelDuringCoronavirusCalculator.new
```

Comment:
Is it possible to stub this? or difficult due to file look up?

Reply:
I don't think this can be stubbed because it's being used to look up the the vaccination status codes that appear in the user response.

Reply:
👍